### ISSUE TITLE
[CAP-1206] CCDB Secret Rotation Command Updates

### ISSUE THREAD

**Issue by btat** [SUSE/doc-cap#675]
From CAP-1206:

The first issue I hit was about missing container names for pods, where within commands you now would also have to specify the container with e.g. "kubectl exec --namespace scf api-group-0 -c api-group".

But finally commands failed for rotating the secrets:

```
kubectl exec --namespace scf api-group-0 -- bash -c 'export PATH=/var/vcap/packages/ruby-2.4/bin:$PATH ; export CLOUD_CONTROLLER_NG_CONFIG=/var/vcap/jobs/cloud_controller_ng/config/cloud_controller_ng.yml ; cd /var/vcap/packages/cloud_controller_ng/cloud_controller_ng ; /var/vcap/packages/ruby-2.4/bin/bundle exec rake rotate_cc_database_key:perform'
```

It refers to ruby-2.4, while the current image contains ruby-2.5.5-r0.10.0.
In addition I failed to bundle install mysql2-0.4.10 (as well as mysql2-0.5.3), saying:

```
Make sure that `gem install mysql2 -v '0.4.10' --source 'https://rubygems.org/'` succeeds before bundling.
```

So I wasn't able to continue with the secret rotation, and I guess the same is true for current CAP-1.5.2-RC2 (2.20.1).

-----

Solution provided:

kubectl exec will default to the first container in the pod, so it was not necessary to specify the container name. You do get an informational message about it though:

```
$ kubectl exec -it api-group-0 --namespace scf /bin/bash
Defaulting container name to api-group.
Use 'kubectl describe pod/api-group-0 -n scf' to see all of the containers in this pod.
api-group/0:/#
 ```

Setting up the PATH and GEMRC has changed; please use this command:

```
source /var/vcap/jobs/cloud_controller_ng/bin/ruby_version.sh
```

This will also make all the GEMs, including mysql2 available automatically.

Finally, you don't need to include the path to bundle in the final command; the tool is now in the first directory on the PATH:

```
bundle exec rake rotate_cc_database_key:perform
```

The command to manually inspect the database also needs amendment because the socket is no longer in the default location. This worked for me:

```
mysql -p${MYSQL_ADMIN_PASSWORD} --socket /var/vcap/sys/run/pxc-mysql/mysqld.sock
```
-----

**Comment by btat** [comment]
This seemed familiar and looking back it looks like the same issues as https://github.com/SUSE/doc-cap/issues/548

**Comment by btat** [comment]
Closed by #696
